packages/panels/src/components/action-menu-label-button/ActionMenuLabelButton.stories.tsx

```diff
@@ -0,0 +1,133 @@
+import { faCheck } from "@fortawesome/free-solid-svg-icons/faCheck";
+import { faFire } from "@fortawesome/free-solid-svg-icons/faFire";
+import { faSave } from "@fortawesome/free-solid-svg-icons/faSave";
+import { Box, Spacing, Text } from "@stenajs-webui/core";
+import * as React from "react";
+import { ActionMenuItem, ActionMenuSeparator } from "@stenajs-webui/elements";
+import { ActionMenuLabelButton } from "./ActionMenuLabelButton";
+import { faCoffee } from "@fortawesome/free-solid-svg-icons/faCoffee";
+
+export default {
+  title: "panels/ActionMenuLabelButton",
+  component: ActionMenuLabelButton,
+};
+
+export const Standard = () => (
```

Comment:
These should be removed:
ActionDropdown
ActionMenuButton

Rename this to ActionMenuButton.
Make label optional.
Add "buttonComponent" prop.
Do not export this publicly.

Add ActionMenuFlatButton, ActionMenuSecondaryButton and ActionMenuPrimaryButton that delegates to ActionMenuButton with buttonComponent prop set.